<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>280</width>
      <height>306</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="plus">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="number1"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="number2"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="result"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="multiply">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="number1_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="number2_2"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="result_2"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="switch_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>切换</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
